metis-enrichment/metis-enrichment-service/pom.xml

```diff
@@ -55,6 +55,17 @@
       <version>${version.mockito.core}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-all</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.core</groupId>
```

Comment:
Good question, it's because of one of the methods throwing BadRequestException which is not present during test..
I'll add a todo, perhaps it's better to change this..